### PR TITLE
revised goals based on feedback

### PR DESCRIPTION
**Pull request by ChristopherMcDonald**
Based on feedback from the Project Goals:
- Change or remove goals that are requirements, or use common metric as opposed to specific design value. For example, “the system will have accuracy of a medium skill level player”, as opposed to “hitting the grid with 75% probability”.
- Stick to goals that are selling points, with explanation why they are good selling points.
- Combine redundant goals(eg. 3 different goals for accuracy)

### CODE REVIEW

File: documentation/projectGoals/ProjectGoals.tex [hunk 1]
```diff
@@ -92,29 +92,29 @@ \section{Goals}
 \subsection{Low-Level Goals}
 The following items encompass the low-level goals of the system:
 \begin{itemize}
-\item The system can detect valid returns from the user with an accuracy of 90\%
-\item The system can hit each square of a 3x3 grid with an accuracy of 75\%
+\item The system can detect valid returns from the user with an accurately
```
Comment (platkis) [inline documentation/projectGoals/ProjectGoals.tex]: fix grammar

File: documentation/projectGoals/ProjectGoals.tex [hunk 2]
```diff
@@ -92,29 +92,29 @@ \section{Goals}
...
+\item The system can hit 9 zones on the table, corresponding to a 3x3 grid
 \item The system implements reinforcement learning algorithms
-\item The system can provide insightful feedback to the user
+\item The system helps user improve their game in any way % made change to sound more like a goal
```
Comment (platkis) [inline documentation/projectGoals/ProjectGoals.tex]: Not big on the way this is worded. something more like "the system should help the user hone their skills"

File: documentation/projectGoals/ProjectGoals.tex (hunk 1 above)
Comment (ChristopherMcDonald) [reply documentation/projectGoals/ProjectGoals.tex]: Will do!

File: documentation/projectGoals/ProjectGoals.tex [hunk 3]
```diff
@@ -92,29 +92,29 @@ \section{Goals}
...
+The above list encompasses what we determine as a successful product. The system would be 
 \subsection{Mid-Level Goals}
 The following items encompass the mid-level goals of the system:
 \begin{itemize}
-\item The system can hit each square of a 4x4 grid with an accuracy of 75\%
-\item The system can notify the user of their performance after each shot is taken
-\item The system has multiple modes including `single-shot' and random
+\item The system can hit 16 zones on the table, corresponding to a 4x4 grid
+\item The system has realtime performance updates
+\item The system has implements the `single-shot' and random shooting modes
```
Comment (platkis) [inline documentation/projectGoals/ProjectGoals.tex]: remove "has"

File: documentation/projectGoals/ProjectGoals.tex [hunk 4]
```diff
@@ -92,29 +92,29 @@ \section{Goals}
...
 \end{itemize}
+The above list encompasses what we determine as a successful product. The system would be 
 \subsection{Mid-Level Goals}
 The following items encompass the mid-level goals of the system:
 \begin{itemize}
-\item The system can hit each square of a 4x4 grid with an accuracy of 75\%
-\item The system can notify the user of their performance after each shot is taken
-\item The system has multiple modes including `single-shot' and random
+\item The system can hit 16 zones on the table, corresponding to a 4x4 grid
+\item The system has realtime performance updates
```
Comment (platkis) [inline documentation/projectGoals/ProjectGoals.tex]: Add dash "real-time"

**Comment by raynerjs** [comment]
I'm happy with this, looks like we made all the changes from the TAs

**Pull request merged**